spectral_cube/cube_utils.py

```diff
@@ -791,16 +792,16 @@ def combine_headers(header1, header2):
     header['WCSAXES'] = 3
     return header
 
-def mosaic_cubes(cubes, spectral_block_size=100, **kwargs):
+def mosaic_cubes(cubes, spectral_block_size=100, combine_header_kwargs={}, **kwargs):
     '''
-    This function reprojects cubes onto a common grid and combines them to a single field.  
+    This function reprojects cubes onto a common grid and combines them to a single field.
 
     Parameters
     ----------
     cubes : iterable
-        Iterable list of SpectralCube objects to reproject and add together. 
+        Iterable list of SpectralCube objects to reproject and add together.
     spectral_block_size : int
-        Block size so that reproject does not run out of memory. 
+        Block size so that reproject does not run out of memory.
 
```

Comment:
Add a docstring for `combine_header_kwargs` on passing these to `reproject.mosaicking.find_optimal_celestial_wcs`

Reply:
```suggestion
combine_header_kwargs : dict
     Keywords passed to `~reproject.mosaicking.find_optimal_celestial_wcs`.
```